stack.py

```diff
@@ -0,0 +1,20 @@
+from linked_list import LinkedList
+
+#Stack inherits from LinkedList class
+class Stack(object):
+
+    def __init__(self, iterable=None):
+        if iterable != None:
+            self._linkedList = LinkedList(iterable)
+        else:
+            self._linkedList = LinkedList()
+
+    def push(self, value):
+        self._linkedList.insert(value)
+
+    def pop(self):
+        try: 
+            return self._linkedList.pop()
+        except AttributeError:
+            print "Error: Stack is empty. Nothing to pop."
+            raise
```

Comment:
this is an odd choice.  On the one hand, I can see intercepting the AttributeError raised by the contained `_linkedList`, but if you do so, you probably want to raise a new exception from here, because the sole purpose of doing so would be to make it look like the source of the error was the `Stack` rather than the containe `_linkedList`.  

Here, you print an error message, and then go ahead and raise the same error you just caught, which leaves the source of the error looking like it's the contained `_linkedList`.  Printing is never good error handling.  And raising the same error back up defeats the purpose of intercepting it in the first place.  

And you're missing a newline character.  Github is yelling at you about it.  Is your linter doing so as well?